sdk/python/feast/feature_view.py

```diff
@@ -259,7 +259,7 @@ def join_keys(self) -> List[str]:
 
     @property
     def schema(self) -> List[Field]:
-        return self.entity_columns + self.features
+        return list(set(self.entity_columns + self.features))
```

Comment:
Does Field implement `__eq__` correctly?

Reply:
just updated it to check all attributes of `Field` properly, so yes I think so